### ISSUE TITLE
No images on shared posts

### ISSUE THREAD

**Issue by rafcontreras** [mikahanninen/grav-plugin-facebook#25]
Hi there,

When a post is a shared post, there's no image to show.

Shared posts have this response, notice the **subattachments**:

```{
    "feed": {
        "data": [{
            "permalink_url": "https://www.facebook.com/...",
            "created_time": "...",
            "link": "https://www.facebook.com/...",
            "message": "...",
            "id": "...",
            "attachments": {
                "data": [{
                    "subattachments": {
                        "data": [{
                            "media": {
                                "image": {
                                    "height": 720,
                                    "src": "https://scontent.xx.fbcdn.net/...",
                                    "width": 719
                                }
                            },
                            "target": {
                                "id": "10155111642401006",
                                "url": "https://www.facebook.com/..."
                            },
                            "type": "photo",
                            "url": "https://www.facebook.com/..."
                        }, {
                            "media": {
                                "image": {
                                    "height": 720,
                                    "src": "https://scontent.xx.fbcdn.net/...",
                                    "width": 350
                                }
                            },
                            "target": {
                                "id": "10155111642396006",
                                "url": "https://www.facebook.com/..."
                            },
                            "type": "photo",
                            "url": "https://www.facebook.com/..."
                        }]
                    },
                    "target": {
                        "id": "...",
                        "url": "..."
                    },
                    "title": "...",
                    "type": "album",
                    "url": "..."
                }]
            }
        }
    ]
}```

**Comment by mikahanninen** [comment]
I have been on vacation for over an week. I will get back to this soon as possible.

**Comment by mikahanninen** [comment]
I think this will be fixed with https://github.com/mikahanninen/grav-plugin-facebook/pull/27

**Comment by mikahanninen** [comment]
Please close if the fix has been confirmed with version 1.6.0

**Comment by mikahanninen** [comment]
Considered as closed due to inactivity.
